routemaster/cron_processors.py

```diff
@@ -21,6 +23,25 @@ def _logger_for_type(type_: Type[Any]) -> logging.Logger:
     return logging.getLogger(f"({type_.__module__}.{type_.__name__}")
 
 
+def _every_minute_to_now(
+    time: datetime.datetime,
+) -> Iterator[datetime.datetime]:
+    now = datetime.datetime.now(dateutil.tz.tzutc())
+    while time <= now:
+        yield time
+        time += datetime.timedelta(minutes=1)
```

Comment:
Minor, but do we want to truncate the time components to be precisely on the minutes?

Reply:
Possibly, though this is redundant since 8ac282aeb190f69dca1c30ddeb8f5194e5804426 (and bade4574957b5216f701833cb8456eba69895bfe) which change the way we account for delays to operate on ranges rather than looping over each minute.